kombu/transport/redis.py

```diff
@@ -147,8 +146,12 @@ def __init__(self, *args, **kwargs):
     def append(self, message, delivery_tag):
         delivery = message.delivery_info
         EX, RK = delivery['exchange'], delivery['routing_key']
-        # Redis-py changed the format of zadd args in v3.0.0 to be like this
-        zadd_args = [{delivery_tag: time()}]
+        # TODO: Remove this once we soley on Redis-py 3.0.0+
+        if redis.VERSION[0] >= 3:
```

Comment:
Yes it does...

Reply:
I will handle this in a pr after merging this